learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+- The `main` function is your program’s entry point, and you can execute it by running the program. 
+- This function contains a statement that prints `Hello, World!` to the console. This statement calls the `println` function in the `io` with `Hello, World!`.
+
+If you are interested in learning more about the language, read our [Language Basics](https://ballerina.io/learn/language-basics/) guide. 
+
+Let’s run this project by running `bal run` in your terminal:
+
+```
+$ bal run
+Compiling source
+	example/greeter:0.1.0
+
+Running executable
+
+Hello, World!
+```
+
+You can also generate an executable program with `bal build`:
+
+```$ bal buildCompiling source	sameera/greeter:0.1.0
+Generating executable	target/bin/greeter.jar
+```
+
+Then, use `bal run` as follows.
+
+```
+$ bal run target/bin/greeter.jar
+Hello, World!
+```
+
+## Write a simple REST API
+
+Now let’s change our greeter application to a REST API. Ballerina has first-class abstractions for services, resources, etc., and they make network service development easier and more fun. 
+
+You can replace the `main.bal` content with the following code.
+
+```
+import ballerina/http;
+
+listener http:Listener httpListener = new (8080);
+
+service / on httpListener {
+    resource function get greeting() returns string { 
+        return "Hello, World!"; 
+    }
+
+    resource function get greeting/[string name]() returns string { 
+        return "Hello " + name; 
+    }
+}
+```
+
+Let’s take a moment to digest new constructs in this code:
+
+- The `[http](https://lib.ballerina.io/ballerina/http/latest)` module provides high-level abstractions to work with the HTTP protocol. 
+- The listener declaration creates a new HTTP listener with port 8080. Listener is the entity that receives the network input and then routes it to the attached service(s).
+- The service declaration specifies the listener, to which the service gets attached, and a collection of remotely accessible methods. There are two kinds of methods: `resource` and `remote` methods.
+- Services use remote methods to expose services in procedural style: remote methods are named by verbs, and resource methods for data-oriented protocols: resources are named by nouns.
+- In this example, there are two resource methods: The first one responds to `HTTP` `GET` requests with the path `/greeting`, and the other one responds to `GET` requests with the path `/greeting/{name}`.
+- These resource methods return a `string` value that maps to the content-type `text/plan` in the `HTTP` response.
+
+
+
+If you are interested in learning more about services, read our [Network Interaction](https://ballerina.io/learn/distinctive-language-features/network-interaction/) guide. 
```

Comment:
```suggestion
If you are interested in learning more about services, read our [Network Interaction](https://ballerina.io/learn/distinctive-language-features/network-interaction/) guide. 
```
Instead of saying `if you are interested,` let's directly say as follows.

To learn more about services, see [Network Interaction](https://ballerina.io/learn/distinctive-language-features/network-interaction/).

Reply:
Fixed